src/scripts/main.js

```diff
@@ -2,7 +2,61 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dash = document.querySelector('.dashboard');
 
-// write your code here
+function addData(arr) {
+  for (let i = 0; i < arr.length; i++) {
+    const newRow = dash.insertRow(i + 1);
+    let personBorn;
+
+    for (const j in arr[i]) {
```

Comment:
this loop redundant, please try to create table row element and aftre that add html with `insertAdjacentHTML `method, it should be smth like this:
tableRow.insertAdjacentHTML('beforeend', `
    <td>${.name}</td>
`)